File: 0001 Code Chef/0025 Burger/implementation/0011 lua/burger.lua
test = tonumber(io.read())
while(test > 0)do
    string_input = io.read()
    string_array = {}
    index = 1;
    for i in string.gmatch(string_input,"%S+") do
        string_array[index] = i
        index = index + 1;
    end
    a = tonumber(string_array[1]);
    b = tonumber(string_array[2]);
    if(a > b) then
        print(b)
    else
        print(a)
    end
    test = test - 1;
end

File: 0001 Code Chef/0025 Burger/implementation/0006 php/burger.php
<?php
$test = (int) fgets(STDIN);
while($test > 0){
    $string_input = fgets(STDIN);
    $string_array = explode(" ",$string_input);
    $a = (int) $string_array[0];
    $b = (int) $string_array[1];
    if($a > $b){
        echo "$b\n";
    }else{
        echo "$a\n";
    }
    $test = $test - 1;
}

?>